synapse/__init__.py

```diff
@@ -25,6 +25,27 @@
     print("Synapse requires Python 3.7 or above.")
     sys.exit(1)
 
+# Allow using the asyncio reactor via env var.
+if bool(os.environ.get("SYNAPSE_ASYNC_IO_REACTOR", False)):
+    try:
+        from incremental import Version
```

Comment:
Should `incremental` be added into `python_dependencies` since we're now using it directly?

Reply:
Possibly, though I was imagining this only being a temporary thing while we had the experimental flag, so not much point adding it to the dependencies?